mappings/net/minecraft/enchantment/EnchantmentLevelEntry.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_1889 net/minecraft/enchantment/EnchantmentLevelEntry
+	COMMENT Represents an entry in an enchantments array, i.e. a pair between
+	COMMENT enchantment and level. This is also a weighted entry and is available
+	COMMENT for the weighted picker to pick from a list.
+	COMMENT
+	COMMENT <p>This is usually used where multiple enchantment to level mappings can
+	COMMENT exist, such as in enchanting logic. In other cases, vanilla prefers
+	COMMENT {@code Map<Enchantment, Integer>} over {@code List<EnchantmentLevelEntry>}.
+	COMMENT
+	COMMENT <p>This class, though effectively immutable, does not override hashCode
+	COMMENT or equals.
```

Comment:
I don't think defining immutability has anything to do with `hashCode` and `equals`.

Reply:
Well, hashCode and equals are unreliable if an object isn't immutbale, i.e. immutability is a prerequisite for useful hashCode/equals

Reply:
Updated to show that these two have no relationship